README.md

```diff
@@ -7,7 +7,7 @@ pydicom
 [![codecov](https://codecov.io/gh/pydicom/pydicom/branch/master/graph/badge.svg)](https://codecov.io/gh/pydicom/pydicom)
 [![Python version](https://img.shields.io/pypi/pyversions/pydicom.svg)](https://img.shields.io/pypi/pyversions/pydicom.svg)
 [![PyPI version](https://badge.fury.io/py/pydicom.svg)](https://badge.fury.io/py/pydicom)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1450666.svg)](https://doi.org/10.5281/zenodo.1450666)
```

Comment:
I usually rebase to master to avoid that mixup with other commits, even if it means a force-push

Reply:
Yeah, I was kinda doing too many things at once.  But I thought this should be no problem since I always squash/merge when merging in PRs?

Reply:
No, I don't think that would be a problem. It is just nicer for the review...